acid/features/history/tests/conftest.py

```diff
@@ -29,31 +40,40 @@ def _make_buildset(build_number, branch='master'):
                                 ref_url='http://acid.test/gitweb/',
                                 oldrev='', newrev='')
         commit()
-        make_build(build_number=build_number, buildset_id=buildset.id)
-        make_build(build_number=build_number, buildset_id=buildset.id)
-        return buildset
+        if (start_time == 0):
+            start_time = datetime(2018, 2, 23, 22, 0, 0)
+        if (end_time == 0):
+            end_time = datetime(2018, 2, 23, 23, 55, 0)
+        if (start_time and end_time):
+            time_diff = (end_time - start_time).total_seconds()
+        else:
+            time_diff = 0
+        starting_times = [start_time]
+        ending_times = [end_time]
```

Comment:
maybe this long function could be divided into smaller functions? It could improve readability with low cost